mmdet/models/losses/cross_entropy_loss.py

```diff
@@ -239,13 +252,26 @@ def forward(self,
                 self.class_weight, device=cls_score.device)
         else:
             class_weight = None
-        loss_cls = self.loss_weight * self.cls_criterion(
-            cls_score,
-            label,
-            weight,
-            class_weight=class_weight,
-            reduction=reduction,
-            avg_factor=avg_factor,
-            ignore_index=ignore_index,
-            **kwargs)
+
+        if self.use_sigmoid and not self.bce_expand_one_hot:
```

Comment:
It is ugly to use `if_else`.
There are `**kwargs`, if you had to add `expand_one_hot` in bce_loss, you can add `**kwargs` to ce loss and mask_ce loss to handle this argument.